src/main/java/com/salesforce/storm/spout/sideline/config/DocTask.java

```diff
@@ -63,7 +63,7 @@ private static void generateMetricDocs(final Path inputPath) throws IOException
     private static void generateConfigDocs(final Path inputPath) throws IOException {
         final String tagArg = "SIDELINE_CONFIGURATION";
         final List<ClassSpec> classSpecs = new ArrayList<>();
-        classSpecs.add(new ClassSpec(SidelineConfig.class, SidelineConfig.setDefaults(new HashMap())));
+        classSpecs.add(new ClassSpec(SidelineConfig.class, SidelineConfig.setDefaults(Maps.newHashMap())));
```

Comment:
and here I was working towards removing the dependency on common collections by removing these :p